src/adminjs.ts

```diff
@@ -39,12 +45,12 @@ export const defaultOptions: AdminJSOptionsWithDefault = {
 }
 
 type ActionsMap = {
-  show: Action<RecordActionResponse>;
-  edit: Action<RecordActionResponse>;
-  delete: Action<RecordActionResponse>;
-  bulkDelete: Action<BulkActionResponse>;
-  new: Action<RecordActionResponse>;
-  list: Action<ListActionResponse>;
+  show: Action<RecordActionResponse>
```

Comment:
❓ Is this some new Eslint rule to remove semicolons?

Reply:
IDK, I have autoformatting enabled on save and it does that